src/lib/Repository/UiRelationService.php

```diff
@@ -62,17 +62,18 @@ public function loadRelations(VersionInfo $versionInfo)
     /**
      * Loads reverse relations for a piece of content.
      * Retrieves reverse relations and then sets the field definition and the content type name.
+     * Returns false if the user doesn't have permission to load reverse relations.
      *
      * @param ContentInfo $contentInfo
      *
-     * @return UiRelation[]
+     * @return UiRelation[]|bool
      */
     public function loadReverseRelations(ContentInfo $contentInfo)
     {
         try {
             $reverseRelations = $this->contentService->loadReverseRelations($contentInfo);
         } catch (UnauthorizedException $e) {
```

Comment:
I'm not sure using a constant is a much better approach. I've updated the PR to check the permission beforehand which I believe is a cleaner approach. Let me know what you think?